_posts/2015-09-15-Nepal-Earthquake-Relief-and-Humanitarian-Aid-Monitoring-System.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Nepal Earthquake Relief and Humanitarian Aid Monitoring System"
+author: alina_acharya
+modified: 2015-09-15
+tags: [earthquake, humanitarian aid, pledge]
+---
+
+In April 25, 2015, earthquake of magnitude 7.8 struck Nepal that destroyed millions of home and slaughtered thousands of people. It became one of the destructive events in Nepalese history after 1934 earthquake.  The constant aftershocks and two major quakes had left Nepalese people in deep sorrow. The earthquake killed more than 8,960 and injured more than twice as many. Many historic buildings were collapsed, one of the greatest historic building “Dharahara” was turned down in pile of mud. More than 600,000 people were displaced form their home. 
```

Comment:
`On April 25...`

`millions of homes and thousands of people lost their lives` --> to make it less graphic 

It became the most destructive events in Nepalese history after 1934 earthquake. (1934 AD or BS)??

“Dharahara” was turned down to a pile of mud and debris.